src/reader/OTF2Reader.cpp

```diff
@@ -399,8 +397,10 @@ OTF2_CallbackCode OTF2Reader::handle_metric(OTF2_LocationRef locationID, OTF2_Ti
                           metricValues[i].floating_point};
                 }
 
-                tmp_metric.insert(make_pair(metric_ref->first, md));
-            }
+                tmp_metric.insert(make_pair(metric_ref->second, md));
+            } else {
```

Comment:
empty else body?

Reply:
can you pleae use clang format? Especially for  create_json.cpp

Reply:
Missed the empty else body when I was cleaning out debug output. Will fix.